python/doc/examples/data_analysis/distribution_fitting/plot_estimate_conditional_quantile.py

```diff
@@ -7,9 +7,9 @@
 # From a multivariate data sample we estimate its distribution with kernel smoothing. Here we present a bivariate distribution  X= (X1, X2). We use the `computeConditionalQuantile` method to estimate the 90% quantile :math:`Q_1` of the conditional variable :math:`X1|X2` :
```

Comment:
X2|X1 here too then (at end of line) ?

Reply:
Yes... it has been included in the proposition of Joseph.